chia/full_node/mempool_check_conditions.py

```diff
@@ -45,6 +58,39 @@ def get_name_puzzle_conditions(
     if height >= constants.SOFT_FORK2_HEIGHT:
         flags = flags | ENABLE_ASSERT_BEFORE | NO_RELATIVE_CONDITIONS_ON_EPHEMERAL
 
+    if height >= constants.SOFT_FORK3_HEIGHT:
+        # the soft-fork initiated with 2.0. To activate end of October 2023
+        # * the number of announces created and asserted are limited per spend
+        # * the total number of CLVM objects (atoms or pairs) are limited
+        # * BLS operators enabled, behind the softfork op. This set of operators
+        #   also include coinid, % and modpow
+        # * secp operators enabled
+        flags = flags | LIMIT_ANNOUNCES | LIMIT_OBJECTS | ENABLE_BLS_OPS | ENABLE_SECP_OPS
+
+    if height >= constants.HARD_FORK_HEIGHT:
+        # the hard-fork initiated with 2.0. To activate June 2024
+        # * costs are ascribed to some unknown condition codes, to allow for
```

Comment:
There is no flag assigned for this bullet point like with all the others but i guess its part of `ENABLE_SOFTFORK_CONDITION`?

Reply:
yes, that's right. it's enabled by that same flag